services/server/src/sourcify-chains-default.json

```diff
@@ -1892,5 +1892,21 @@
     "sourcifyName": "TixChain Testnet",
     "supported": true,
     "rpc": ["https://subnets.avax.network/tixchain/testnet/rpc"]
+  },
+  "223": {
+    "sourcifyName": "B2 Mainnet",
+    "supported": true,
+    "etherscanApi": {
+      "apiURL": "https://explorer.bsquared.network/api"
+    },
+    "fetchContractCreationTxUsing": {
+      "blockscoutApi": {
+        "url": "https://mainnet-backend.bsquared.network"
+      },
+      "routescanApi": {
+        "type": "mainnet"
+      },
```

Comment:
I cannot find your network on https://routescan.io/. Is Routescan really supporting it?